contrib/src/templates/tera_templates.rs

```diff
@@ -8,7 +8,14 @@ use super::{TemplateInfo, TEMPLATE_DIR};
 lazy_static! {
     static ref TERA: Result<tera::Tera, String> = {
         let path: PathBuf = [&*TEMPLATE_DIR, "**", "*.tera"].iter().collect();
-        tera::Tera::new(path.to_str().unwrap()).map_err(|e| format!("{:?}", e))
+        let escape_ext = [".html.tera", ".htm.tera", ".xml.tera",
```

Comment:
Yeah, 85 is arbitrary. Rust's default cutoff is 100.

Definitely not optimizing for line count. But, things that are concise are sometimes easier to read. I think that's the case here.